lib/yearbook/university.ex

```diff
@@ -295,4 +295,33 @@ defmodule Yearbook.University do
   def change_class(%Class{} = class, attrs \\ %{}) do
     Class.changeset(class, attrs)
   end
+
+  alias Yearbook.University.ClassStudent
+
+  def create_class_student(attrs \\ %{}) do
+    student = list_classes_student(attrs["student_id"], attrs["class_id"])
+
+    if student > 0 do
```

Comment:
These types of verifications should be at the database level. Otherwise, you can not guaranty it.

Just make sure you have the unique index of [:student_id, :class_id] :muscle: